packages/eddsa-ticket-pcd/src/utils.ts

```diff
@@ -13,6 +13,10 @@ export type SerializedTicket = [
   bigint,
   bigint,
   bigint,
+  bigint,
+  bigint,
```

Comment:
nit: Now that these aren't public in the ticket, there's no documentation explaining what the extra fields are for.  A block comment here where they're included in the signature might be the way to document how they should be used.